sphinx_js/templates/common.rst

```diff
@@ -0,0 +1,30 @@
+{% macro deprecated(message) %}
+{% if message -%}
+.. note::
+
+   Deprecated
+   {%- if message is string -%}: {{ message }}{% else %}.{% endif -%}
```

Comment:
Shouldn't this be just `if message`? You default in the `.get()` call is `''`, so it will always be a string. Otherwise, looks great.

Reply:
Ah what happens is if you just have `@deprecated` in JSDoc, then the doclet contains `deprecated: True`. So we have to check if the field is actually a string before rendering it.